commons/connector/health/aggregation/health_blood_measurements.avsc

```diff
@@ -0,0 +1,49 @@
+{
+  "namespace": "org.radarcns.active.healthkit",
+  "name": "HealthKitBloodMeasurements",
+  "type": ["null", "record"],
+  "default": null,
+  "doc": "blood_glucose, blood_pressure_diastolic, blood_pressure_systolic, oxygen_saturation, vo2max, insulin",
+  "fields": [
+    {
+      "name": "time",
+      "type": "double",
+      "doc": "Device timestamp in UTC (s)"
+    },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "Time that the data was received from the Apple Healthkit"
+    },
+    {
+      "name": "blood_pressure_systolic",
```

Comment:
Use camel case (`bloodPressureSystolic`)

Reply:
There are _ and . for naming because when we are getting the info from cordova-health, it will need this particular name to get the info, so If we want to use different value in our schema, I have to write a lot of  (if else) to judge the input, is there any constraints on schema to use _ or . ?